src/bufr/CMakeLists.txt

```diff
@@ -3,30 +3,73 @@
 
-#list(APPEND _ingester_srcs
-#)
+list(APPEND _ingester_srcs
+  IngesterTypes.h
+  DataContainer.h
+  DataContainer.cpp
+  Parser.h
+  ParserFactory.h
+  DataObject/DataObject.h
+  DataObject/StrVecDataObject.h
+  DataObject/StrVecDataObject.cpp
+  DataObject/ArrayDataObject.h
+  DataObject/ArrayDataObject.cpp
+  BufrParser/BufrParser.h
+  BufrParser/BufrParser.cpp
+  BufrParser/BufrDescription.h
+  BufrParser/BufrDescription.cpp
+  BufrParser/BufrMnemonicSet.h
+  BufrParser/BufrMnemonicSet.cpp
+  BufrParser/BufrTypes.h
+  BufrParser/BufrCollectors/BufrAccumulator.cpp
+  BufrParser/BufrCollectors/BufrAccumulator.h
+  BufrParser/BufrCollectors/BufrCollector.cpp
+  BufrParser/BufrCollectors/BufrCollector.h
+  BufrParser/BufrCollectors/BufrIntCollector.cpp
+  BufrParser/BufrCollectors/BufrIntCollector.h
+  BufrParser/BufrCollectors/BufrRepCollector.cpp
+  BufrParser/BufrCollectors/BufrRepCollector.h
+  BufrParser/BufrCollectors/BufrCollectors.cpp
+  BufrParser/BufrCollectors/BufrCollectors.h
+  BufrParser/Exports/Export.h
+  BufrParser/Exports/DatetimeExport.h
+  BufrParser/Exports/DatetimeExport.cpp
+  BufrParser/Exports/MnemonicExport.h
+  BufrParser/Exports/MnemonicExport.cpp
+  BufrParser/Exports/Transforms/Transform.h
+  BufrParser/Exports/Transforms/OffsetTransform.h
+  BufrParser/Exports/Transforms/OffsetTransform.cpp
+  BufrParser/Exports/Transforms/ScalingTransform.h
+  BufrParser/Exports/Transforms/ScalingTransform.cpp
+  BufrParser/Exports/Transforms/TransformBuilder.h
+  BufrParser/Exports/Transforms/TransformBuilder.cpp
+  IodaEncoder/IodaEncoder.cpp
+  IodaEncoder/IodaEncoder.h
+  IodaEncoder/IodaDescription.cpp
+  IodaEncoder/IodaDescription.h
+)
 
-#list(APPEND INGESTER_LIBS_DEP
-#  Eigen3::Eigen
-#  eckit
-#  oops
-#  ioda-engines
-#  bufr::bufr_4_DA)
+list(APPEND _ingester_deps
+  Eigen3::Eigen
+  eckit
+  ${oops_LIBRARIES}
+  ioda_engines
+  bufr::bufr_4_DA
+)
 
-#ecbuild_add_library( TARGET   ingester
-#                     SOURCES  ${_ingester_srcs}
-#                     LIBS     ${INGESTER_LIBS_DEP}
-#                     INSTALL_HEADERS LISTED
-#                     HEADER_DESTINATION ${INSTALL_INCLUDE_DIR}/iodaconv
-#                     LINKER_LANGUAGE ${IODACONV_LINKER_LANGUAGE}
-#                    )
+ecbuild_add_library( TARGET   ingester
+                     SOURCES  ${_ingester_srcs}
+                     LIBS     ${_ingester_deps}
+                     INSTALL_HEADERS LISTED
+                     HEADER_DESTINATION ${INSTALL_INCLUDEDIR}/bufr
+                     LINKER_LANGUAGE CXX
+                    )
+add_library( ${PROJECT_NAME}::ingester ALIAS ingester )
 
-#target_link_libraries(ingester PRIVATE oops::oops)
-#target_link_libraries(ingester PRIVATE ioda_engines::ioda_engines)
+ecbuild_add_executable( TARGET  convertToIoda.x
+                        SOURCES main.cpp
+                        LIBS    ingester )
+#target_link_libraries(convertToIoda.x PRIVATE ${PROJECT_NAME}::ingester)
```

Comment:
whoops, this is probably my fault, but this commented out line should be removed